x-pack/plugins/rule_registry/server/routes/get_alert_summary.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Boom from '@hapi/boom';
+import { IRouter } from '@kbn/core/server';
+import * as t from 'io-ts';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import moment from 'moment';
+
+import { RacRequestHandlerContext } from '../types';
+import { BASE_RAC_ALERTS_API_PATH } from '../../common/constants';
+import { buildRouteValidation } from './utils/route_validation';
+
+export const getAlertSummaryRoute = (router: IRouter<RacRequestHandlerContext>) => {
+  router.post(
+    {
+      path: `${BASE_RAC_ALERTS_API_PATH}/_alert_summary`,
+      validate: {
+        body: buildRouteValidation(
+          t.intersection([
+            t.exact(
+              t.type({
+                gte: t.string,
+                lte: t.string,
+              })
+            ),
+            t.exact(
+              t.partial({
+                featureIds: t.array(t.string),
+                fixed_interval: t.string,
+                filter: t.object,
+                index: t.string,
+              })
+            ),
+          ])
+        ),
+      },
+      options: {
+        tags: ['access:rac'],
+      },
+    },
+    async (context, request, response) => {
+      try {
+        const racContext = await context.rac;
+        const alertsClient = await racContext.getAlertsClient();
+        const { gte, lte, featureIds, filter, fixed_interval: fixedInterval, index } = request.body;
+        if (
+          !(
+            moment(gte, 'YYYY-MM-DDTHH:mm:ss.SSSZ', true).isValid() &&
+            moment(lte, 'YYYY-MM-DDTHH:mm:ss.SSSZ', true).isValid()
+          )
+        ) {
+          throw Boom.badRequest('gte and/or lte are not following the UTC format');
+        }
+
+        if (fixedInterval && fixedInterval?.match(/^\d{1,2}['m','h','d','w']$/) == null) {
```

Comment:
Should we validate this with a custom validator on the body params?

Reply:
I did not find an easy way to do it with io-ts, so I will keep it this way for now.